packages/yew-macro/tests/html_macro_test.rs

```diff
@@ -22,6 +24,7 @@ fn dynamic_tags_catch_void_elements() {
 }
 
 #[test]
+#[cfg(feature = "lints")]
```

Comment:
I think you forgot a `not` here ;)

But actually, I think it would better to remove these `cfg` attributes altogether and instead use `cargo test html_lints` to only run the specific tests for lints.
It makes sense to keep the `cfg(feature = "lints")` on `test_html_lints` though.

Reply:
Yeah that makes more sense